### PR TITLE
Show token cache usage

### PR DESCRIPTION
**Pull request by skabou**
Show token cache usage

### CODE REVIEW

**Comment by skabou** [comment]
@ckittel Curious about your thoughts on this

image

File: msal-client-credentials-flow/app.js [hunk 1]
```diff
@@ -48,6 +48,10 @@ app.get('/api/application', (req, res) => {
       headers: { Authorization: `Bearer ${response.accessToken}` }
     }
 
+    // MSAL Node uses an in-memory token cache by default
+    // Repeated HTTP GET requests will utilize the cached token until it nears expiration, at which point a new access token will be requested.
+    console.log(`\n\nToken: ${response.accessToken}`)
+
```
Comment (ckittel) [inline msal-client-credentials-flow/app.js:54]: Wouldn't it be better to log based on `response.fromCache`?  No need to dump the whole thing.  You can build a log statement that says it if was from cache or newly acquired.

**Review by ckittel — CHANGES_REQUESTED** [review]

File: msal-client-credentials-flow/app.js (hunk 1 above)
Comment (skabou) [reply msal-client-credentials-flow/app.js:54]: Absolutely, great idea

**Review by ckittel — APPROVED** [review]
:shipit:

**Pull request merged**